### ISSUE TITLE
Create an accessibility section in JupyterLab documentation

### ISSUE THREAD

**Issue by steff456** [jupyterlab/jupyterlab#14396]
### Problem

- We don't have any documentation that is accessibility-focused in the JupyterLab documentation Given that the community is including accessibility-focused improvements, it is becoming increasingly necessary that we provide a dedicated focus on adding one page for user accessibility documentation for JupyterLab.

### Proposed Solution

- Create an accessibility section in JupyterLab documentation containing default accessibility settings and customizations inspired by https://code.visualstudio.com/docs/editor/accessibility

**Comment by steff456** [comment]
> This is a single page, right?

Yes! It is a single page

> Perhaps that should be added as a note at the top of the page? Not sure how to message it.

We can add it like the dates that are in the accessibility documentation :)

**Comment by krassowski** [comment]
I see that accessibility section is in there: https://jupyterlab.readthedocs.io/en/latest/getting_started/accessibility.html

It looks a bit outdated, showing 2022 date and JupyterLab 3.4.4. Are there plans to update it going forward?

**Comment by gabalafou** [comment]
The accessibility statement and this issue are separate concerns.

This issue was about creating a help section for a disabled end-user audience that explains various accessibility-related tips and tricks for JupyterLab.

The accessibility statement is more of a formal document about the overall state of JupyterLab conformance to WCAG. 

Unfortunately, the help section work had to be abandoned.

I re-read the accessibility statement just now, and to my eyes, apart from the reference to the older version of JupyterLab, I'm not sure much needs to be changed. I created a pull request to make those changes: #16206.
